typing_tester.py

```diff
@@ -0,0 +1,114 @@
+import curses
+from curses import wrapper
+import time
+
+# To calcukate accuracy
+def cal_accuracy(given_word, user_text, backspaces):
+    wrong_letter = 0
+    try:
+        for i in range(len(given_word)):
+            if user_text[i] != given_word[i]:
+                wrong_letter = wrong_letter + 1
+    except:
+        wrong_letter = wrong_letter+1
+
+    total_mistakes = wrong_letter + backspaces
+    accuracy = len(given_word) - total_mistakes
+    accuracy =  accuracy/len(given_word)*100
+    return float("{:.2f}".format(accuracy))
+
+# To calculate Time Score
+def cal_time(elapsed_time, given_word):
+    raw_score = elapsed_time / (0.4 + (0.28 * (len(given_word) + 1)))
```

Comment:
Assign these values to some variables or write some kind of comments so they are understandable